src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASIC_COST_PER_DAY = days * 40;
```

Comment:
The name 'BASIC_COST_PER_DAY' is misleading because the value represents the total cost for the number of days, not the cost per day. Consider renaming it to 'totalCost' or a similar name that accurately reflects its purpose.